blockbase/inc/customizer/wp-customize-fonts.php

```diff
@@ -491,23 +491,6 @@ function handle_customize_save_after( $wp_customize ) {
 			$font_families
 		);
 
-		//If the typeface choices === the default then we remove it instead
-		if ( $body_value === $body_default && $heading_value === $heading_default ) {
-			unset( $user_theme_json_post_content->settings->typography->fontFamilies );
```

Comment:
We definitely can't remove this line, its needed to reset the font settings back to default.